app/src/main/res/layout/activity_login.xml

```diff
@@ -2,10 +2,15 @@
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/container"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="#DADDD8"
```

Comment:
Please use the colour values as mentioned in [this](https://github.com/00Chaotic/nApp/pull/9#discussion_r875844624) comment